### PR TITLE
fix(k8s): run k8s test on the latest version of k8s and minikube

### PR DESCRIPTION
**Pull request by kiblik**
Run k8s on the latest version of k8s and minikube, as described in the comment.

### CODE REVIEW

File: .github/workflows/k8s-tests.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
           # are tested (https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#available-versions)
           - databases: pgsql
             brokers: redis
-            k8s: 'v1.26.11'
+            k8s: 'v1.30.3'
```
Comment (cneill) [inline .github/workflows/k8s-tests.yml:31]: I notice minikube is on [`v1.33.1`](https://github.com/kubernetes/minikube/releases/tag/v1.33.1) now. We should probably try to bump that as well (line 40).

Comment (kiblik) [reply .github/workflows/k8s-tests.yml:31]: I agree. Done

**Review by mtesauro — APPROVED** [review]
Approved

**Review by cneill — APPROVED** [review]
Thanks!

**Review by grendel513 — APPROVED** [review]

**Pull request merged**